services/frontend/src/components/EmailInput/EmailInput.scss

```diff
@@ -0,0 +1,6 @@
+@import "styles/_colors.scss";
+@import "styles/_dimens.scss";
+@import "styles/_mixins.scss";
+
+.email-input {
```

Comment:
Could remove the empty selector/the file in general since it's not applying any style changes.